manifests/profile/haproxy.pp

```diff
@@ -52,4 +52,11 @@
     notify  => Service['haproxy'],
   }
 
+  nebula::authzd_user { $monitoring_user['name']:
+    gid  => 'haproxy',
+    home => $monitoring_user['home'],
+    key  => merge($monitoring_user['key'],
+{ command => '/usr/sbin/haproxyctl' } )
```

Comment:
Hm yeah I see what you mean. It might be clearer if it's on one line, and maybe using `+` instead of `merge`, so it can't be confused with the parameters?

```
key => $monitoring_user['key'] + {'command' => '/usr/sbin/haproxyctl'},
```

Reply:
Done.